deny.toml

```diff
@@ -58,12 +58,19 @@ highlight = "all"
 # For each duplicate dependency, indicate the name of the dependency which
 # introduces it.
 # spell-checker: disable
-skip = []
+skip = [
+    { name = "windows-sys" },
```

Comment:
Could you write this like we had it some time ago with the name, version and direct dependency that causes the duplicate? See this old commit for how it was: https://github.com/uutils/coreutils/blob/fd4295398455999f859f05cd098146f9145b2346/deny.toml